src/checkpoint/dispatch/vrt/object_registry.h

```diff
@@ -93,12 +107,12 @@ Registrar<ObjT>::Registrar() {
   debug_checkpoint("object registrar: %zu, %s\n", reg.size(), typeid(ObjT).name());
 
   reg.emplace_back(
-    std::make_tuple(
+    ObjectEntry<BaseType>{
       index,
       []()          -> std::size_t { return sizeof(ObjT); },
```

Comment:
Couldn't this just be `size_t`, evaluated right here?

Reply:
Yep, no reason for that to be a lambda. Currently, this overload isn't even called, but I'm imagining a future variant where the user gets to control allocation

Reply:
Done